centreon/src/Core/Module/Infrastructure/Repository/DbReadModuleInformationRepository.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * Copyright 2005 - 2025 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\Module\Infrastructure\Repository;
+
+use Adaptation\Database\Connection\Collection\QueryParameters;
+use Adaptation\Database\Connection\ValueObject\QueryParameter;
+use Core\Common\Infrastructure\Repository\DatabaseRepository;
+use Core\Module\Application\Repository\ModuleInformationRepositoryInterface;
+use Core\Module\Domain\ModuleInformation;
+
+class DbReadModuleinformationRepository extends DatabaseRepository implements ModuleInformationRepositoryInterface
+{
+    public function findByName(string $name): ?ModuleInformation
+    {
+        $query = $this->queryBuilder
+            ->select('name', 'rname', 'mod_release')
+            ->from('modules_informations')
+            ->where($this->queryBuilder->expr()->equal('name', ':name'))
+            ->getQuery();
```

Comment:
Usage of query builder when there's no dynamic query building is a matter of taste.
You may prefer having your SQL query in one block or keep it like this and both would be ok.

But the usage of the expr inside the where instead of a simple `name = :name` is IMHO less readable.